### PR TITLE
Direct Message Status-Monitor Output to Slack User

### PR DESCRIPTION
**Pull request by jandersson**

### CODE REVIEW

File: profiles/base.yaml [hunk 1]
```diff
@@ -436,4 +436,6 @@ quit_on_status_warning: false
 
 compost_room: 991
 
-carving_workorder_material_type: stone
\ No newline at end of file
+carving_workorder_material_type: stone
+
+slack_username:
```
Comment (jandersson) [inline profiles/base.yaml]: As an example, for my characters it is
`slack_username: jonas`

**Comment by jandersson** [comment]
If you all feel this is too niche, let me know! I won't be offended if the consensus is that this should be a private application. I felt that this was a simple solution to do push notifications. There is potential to respond to the slackbot to have it perform a specified action, e.g. respond with 'remove armband' and 'lick armband'

**Comment by jandersson** [comment]
Do you think it would be better to add the slack_username as an input somewhere in the GUI?

File: status-monitor.lic [hunk 2]
```diff
@@ -154,6 +154,31 @@ class GameFilter
   end
 end
 
+require 'slack-ruby-client'
```
Comment (rcuhljr) [inline status-monitor.lic]: Could you move the slackbot into a separate file? we could then just custom_require it if the slack_username setting exists that way people wanted to use status monitor don't need the gem. So all we'd have here is
```

if get_settings['slack_username']
  custom_require.call('somefile')
  bot = SlackBot.new
end
```

**Comment by jandersson** [comment]
Moved it to a separate file, wondering if there should be a check for script trustedness to avoid this kind of messaging:
```
>;status
>
--- Lich: status-monitor active.
--- Lich: slackbot active.
--- Lich: error: Insecure operation - require
	/home/jonas/.rvm/rubies/ruby-2.0.0-p648/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	/home/jonas/.rvm/rubies/ruby-2.0.0-p648/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:54:in `require'
--- Lich: review this script (slackbot) to make sure it isn't malicious, and type ;trust slackbot
--- Lich: slackbot has exited.
>
```

**Comment by rcuhljr** [comment]
Yes it's going to require trust just like status monitor does.

**Pull request merged**